CHANGELOG.md

```diff
@@ -17,6 +18,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 ### Deprecated
 ### Removed
+- Removed fixture lines that contain passwords/secrets ([654](https://github.com/opensearch-project/opensearch-js/pull/645))
```

Comment:
This reads like a security problem but it's not. If you want a CHANGELOG line, I would extract it into a separate PR. Otherwise I would not even mention it, or get more specific and add that these are test fixtures and that they are incorrectly flagged, e.g. "Removed test fixtures incorrectly flagged by git-secrets for containing passwords/secrets".